### DISCUSSION TITLE
User management page for admins

### DISCUSSION THREAD

**Discussion by cr101** [epicweb-dev/epic-stack#400] [Q&A]
Hi all,

I'm a React beginner and I need to add a page to my app for user management which admin users have access to in order to create/update/delete users of the app as well as handle user password resets.
@kentcdodds do you have plans to add such functionality to the Epic Stack? If not so, has anyone added this functionality to their app that they would like to share please?

**Comment by kentcdodds** [comment]
I expect this functionality will be added to the stack eventually, but it's not in the plans currently.

**Comment by vinstah** [comment]
The users route show them all, I'm unsure if you can edit or change password, one possibility is use the [user-impersonation](https://github.com/alan2207/epic-stack-with-user-impersonation) example
